e3nn/message_passing.py

```diff
@@ -0,0 +1,22 @@
+import torch
+from torch_geometric.nn import MessagePassing
+
+
+class E3Conv(MessagePassing):
+    def __init__(self, Kernel, Rs_in, Rs_out):
+        super(E3Conv, self).__init__(aggr='add')
```

Comment:
```suggestion
        super().__init__(aggr='add')
```
for uniformity with the rest of the code